kore/src/Kore/Attribute/Pattern/Functional.hs

```diff
@@ -26,7 +27,7 @@ import           Kore.Syntax
 {- | A pattern is 'Functional' if it matches exactly one element.
  -}
 newtype Functional = Functional { isFunctional :: Bool }
-    deriving (Eq, GHC.Generic, Show)
+    deriving (Eq, GHC.Generic, Ord, Show)
```

Comment:
What is the `Ord` instance for?

Reply:
Well, it's for testing. To be precise, it's for this `With` instance:
https://github.com/kframework/kore/blob/master/kore/test/Test/Kore/With.hs#L322
used here:
https://github.com/kframework/kore/blob/master/kore/test/Test/Kore/Attribute/Pattern/Functional.hs#L62